src/api/goods.ts

```diff
@@ -10,10 +10,14 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods)
+    .then(goods => [...goods]
+      .sort((good1, good2) => good1.name.localeCompare(good2.name))
+      .slice(0, 5));
```

Comment:
it's unnecessary to use spread operator here since the sort() method can sort the original array in place. Creating a copy of the array can also have performance implications if the array is large. I think it's better to implement this in another way:
 `.then(goods => (
      goods.sort((a, b) => a.name.localeCompare(b.name)).slice(0, 5)
    ));`

Reply:
Good point! Thank you)